plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -535,7 +540,7 @@ function( $feature_id ) use ( $features ) {
 				return $features[ $feature_id ]['is_experimental'];
 			}
 		);
-		$mature_feature_ids       = array_diff( $feature_ids, $experimental_feature_ids );
+		$mature_feature_ids       = array_diff( $feature_ids, $experimental_feature_ids, array( 'cart_checkout_blocks' ) );
```

Comment:
While this works for what you intend it to, I feel we should have a more generic approach to adding features that don't need (or want) to expose a setting in the UI. For example, similar to the `is_experimental` flag we could have a `disable_ui` flag or something like that and then we would exclude those settings from `$feature_ids` below.

I'd be more than happy to review that change if you want to include it in this PR. If not, that's also fine, and we can add it later via a different PR.

Let me know what your thoughts are.

Reply:
This is an awesome suggestion! 💯 Thank you, @jorgeatorres! 🙌